libsolidity/analysis/ImmutableValidator.cpp

```diff
@@ -280,3 +310,11 @@ void ImmutableValidator::visitCallableIfNew(Declaration const& _declaration)
 	if (m_visitedCallables.emplace(_callable).second)
 		_declaration.accept(*this);
 }
+
+
+bool ImmutableValidator::mustHaveBeenWrittenInCurrentControlFlow(VariableDeclaration const& _variableReference) const {
```

Comment:
```suggestion
bool ImmutableValidator::writtenInCurrentControlFlow(VariableDeclaration const& _variableReference) const {
```

This name seems easier to read to me and conveys the same information

Reply:
Done.